developers/bindings/thing-xml.md

```diff
@@ -220,8 +220,13 @@ The following snippet shows a 'Control' and 'Light' tags definition:
 </tags>
 ```
 
-Please note that only [Point](https://openhab.org/javadoc/latest/org/openhab/core/semantics/model/point/package-summary.html) and/or [Property](https://openhab.org/javadoc/latest/org/openhab/core/semantics/model/property/package-summary.html) tags from the pre-defined tag library should be used.
-See also the [Semantic Model]({{base}}/tutorial/model.html) for more information on how tags are used with items.
+There are two possible types of tag -- namely "Semantic" and "Non Semantic" tags.
+The former are used by the openHAB User Interface to create automatic groupings of items according to the [Semantic Model]({{base}}/tutorial/model.html).
+The latter may (optionally) be used for any other tagging purpose at the discretion of the developer or the user.
+The list of Semantic tags is [here](https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.semantics/model/SemanticTags.csv).
```

Comment:
```suggestion
The list of pre-defined Semantic tags is [here](https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.semantics/model/SemanticTags.csv).
```

Reply:
See my above comment about how developers should add non (yet) pre-defined semantic tags.